<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crack</class>
 <widget class="QWidget" name="crack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="captureListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="captureLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QLabel" name="dictionaryLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dictionary</string>
   </property>
  </widget>
  <widget class="QListWidget" name="dictionaryListWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="captureInformationSimpleTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>601</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="captureInformationLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture Information:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addDictionaryButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteDictionaryButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="cout">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savePasswordButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reloadCapturesButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
